src/main/java/wolox/training/models/User.java

```diff
@@ -64,6 +65,10 @@
     @Getter
     private String username;
 
+    @Getter
+    @Setter
+    private String password;
```

Comment:
add `@JsonProperty(access = Access.WRITE_ONLY)`, to not show the password when the user is consulted

Reply:
I'll apply it, thanks.